src/caffeine/caffeine.c

```diff
@@ -56,12 +59,13 @@ void caf_caffeinate(mspace* symmetric_heap, intptr_t* symmetric_heap_start, mspa
   size_t symmetric_heap_size = total_heap_size - non_symmetric_heap_size;
   intptr_t non_symmetric_heap_start = *symmetric_heap_start + symmetric_heap_size;
 
-  if (caf_this_image(*initial_team) == 1) {
+  if (caf_this_image(myworldteam) == 1) {
     *symmetric_heap = create_mspace_with_base(*symmetric_heap_start, symmetric_heap_size, 0);
     mspace_set_footprint_limit(*symmetric_heap, symmetric_heap_size);
   }
   *non_symmetric_heap = create_mspace_with_base(non_symmetric_heap_start, non_symmetric_heap_size, 0);
   mspace_set_footprint_limit(*non_symmetric_heap, non_symmetric_heap_size);
+  *initial_team = &myworldteam;
```

Comment:
I suspect this incorrect assignment motivated the inadvisable change above. I believe this should instead have been:
```suggestion
  *initial_team = myworldteam;
```
I'd be surprised if this version in the current PR commit ran correctly at all...